katas/content/superposition/hardy_state/index.md

```diff
@@ -0,0 +1,3 @@
+**Input:** 2 qubits in the $|00\rangle$ state.
+
+**Goal:** Change the state of the qubits to $\frac{1}{\sqrt{12}} \big(3|00\rangle + |01\rangle + |10\rangle + |11\rangle \big)$.
```

Comment:
The `\ket` macro is also supported in both MathJax and KaTeX engines, and this can simplify the LaTeX and avoid the need for all the `|` and `\rangle` tokens. (See https://github.com/microsoft/qsharp/wiki/Markdown-and-LaTeX-guidelines ).

Reply:
We'll need to do this change uniformly across a lot of code, and I think we'll need to automate this, so I wouldn't block this particular PR for its sake.

Reply:
I have created an issue here to track that change: https://github.com/microsoft/qsharp/issues/1448